src/components/gltf-model-plus.js

```diff
@@ -407,8 +414,16 @@ AFRAME.registerComponent("gltf-model-plus", {
         }
       });
 
+      this.el.object3D.traverse(obj => {
+        obj.frustumCulled = false;
+      });
       this.el.emit("model-loaded", { format: "gltf", model: this.model });
+      await AFRAME.scenes[0].systems.ticker.nextTick();
+      this.el.object3D.traverse(obj => {
+        obj.frustumCulled = true;
+      });
```

Comment:
It still kind of freaks me out that we're blowing away the not-frustum-culledness of anything that was not frustum culled. I would retain the previous setting on the objects (probably on an `{id: setting}` mapping in this code) and restore it afterward.